fastly/validators.go

```diff
@@ -27,6 +27,13 @@ func validateLoggingPlacement() schema.SchemaValidateFunc {
 	}, false)
 }
 
+func validateLoggingServerSideEncryption() schema.SchemaValidateFunc {
+	return validation.StringInSlice([]string{
+		string(gofastly.S3ServerSideEncryptionAES),
+		string(gofastly.S3ServerSideEncryptionAES),
```

Comment:
Looks like a copy paste error

```suggestion
		string(gofastly.S3ServerSideEncryptionKMS),
```